cli/azd/pkg/auth/manager.go

```diff
@@ -263,10 +270,28 @@ func newCredentialFromClientAssertion(
 	return cred, nil
 }
 
-func (m *Manager) LoginInteractive(ctx context.Context, redirectPort int) (azcore.TokenCredential, error) {
+type LoginInteractiveOptions struct {
+	RedirectPort int
+	TenantId     string
+}
+
+func (m *Manager) LoginInteractive(
```

Comment:
NIT: Could we make `nil` for `LoginInteractiveOptions` be a valid option (and mean the default value, similar to methods in the Auzre SDK)